scripts/manage.py

```diff
@@ -0,0 +1,90 @@
+import csv
+import argparse
+import time
+from datetime import datetime
+import getpass
+import math
+
+from web3 import Web3, HTTPProvider
+from solc import compile_files
+
+from utils import CSV_ROWS, get_csv_file_row, wait_for_tx
+
+ADDRESSES_PER_TX = 125
+
+w3 = Web3(HTTPProvider('https://rinkeby.infura.io/ukF8hAZE8Qa0fJIMRCRj'))
+
+
+def get_token_instance(compiled_source, token_address):
+    token_interface = compiled_source['../contracts/Opet_coin.sol:OpetToken']
+    token_contract = w3.eth.contract(
+        abi=token_interface['abi'],
+        bytecode=token_interface['bin'])
+    token_instance = token_contract(token_address)
+    return token_instance
+
+
+def send_airdrop(token_instance, account, drop_file_path):
+    with open(drop_file_path, 'r') as csv_file:
+        addresses, amounts = list(zip(*csv.reader(csv_file)))
+        addresses, amounts = list(addresses), list(amounts)
+        amounts = [int(am) for am in amounts]
+
+    number_of_iterarions = math.ceil(len(addresses) / ADDRESSES_PER_TX)
+    with open('succesfuly_sent.csv', 'w') as csvfile:
+        spamwriter = csv.writer(csvfile)
+        spamwriter.writerow(['Address'])
```

Comment:
Why is it called `spamwriter`? Is it to track the addresses we have sent the airdrop to? It could be better to name it accordingly then (for example, `processed_addresses` or `successfuly_sent_csv` according to the file name).